MediumPower/chains.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "MediumPower",
+  "chains": [
+	{
+      "name": "meme",
+      "address": "mmemevaloper10guelklwjure8zwyrmhaslxf73jn5ur8rzphq5",
```

Comment:
Is this correct? Looks like an extra m at the start

Reply:
Fixed and pushed thank you